### ISSUE TITLE
generate readme programatically

### ISSUE THREAD

**Issue by Zamiell** [gajus/eslint-plugin-jsdoc#1266]
i think that the readme should be generated automatically by a script, so that bugs like #1263 will never happen again.
for reference, you can see my eslint plugin readme generator here: https://github.com/IsaacScript/isaacscript/blob/main/packages/eslint-plugin-isaacscript/scripts/generateReadme.mts

**Comment by brettz9** [comment]
Yeah, would be good to add it to our `generateRule` script, thanks.

**Comment by JoshuaKGoldberg** [comment]
How about onboarding to [`eslint-doc-generator`](https://github.com/bmish/eslint-doc-generator)? It's pretty commonly used in the ecosystem, and quite handy.
